static/js/reducers/videoDetailUi_test.js

```diff
@@ -27,23 +28,27 @@ describe('videoDetailUi', () => {
     sandbox.restore();
   });
 
-  it('should clear the ui', () => {
+  it('should clear the edit dialog ui', () => {
     store.dispatch(setTitle("something"));
-    store.dispatch(clearDialog());
+    store.dispatch(clearEditDialog());
     assert.deepEqual(store.getState().videoDetailUi, INITIAL_UI_STATE);
   });
 
   it('sets the title', () => {
-    assertReducerResultState(setTitle, ui => ui.dialog.title, '');
+    assertReducerResultState(setTitle, ui => ui.editDialog.title, '');
   });
 
   it('sets the description', () => {
-    assertReducerResultState(setDescription, ui => ui.dialog.description, '');
+    assertReducerResultState(setDescription, ui => ui.editDialog.description, '');
   });
 
 
-  it('sets the dialog visibility', () => {
-    assertReducerResultState(setDialogVisibility, ui => ui.dialog.visible, false);
+  it('sets the edit dialog visibility', () => {
+    assertReducerResultState(setEditDialogVisibility, ui => ui.editDialog.visible, false);
+  });
+
+  it('sets the embed dialog visibility', () => {
+    assertReducerResultState(setShareDialogVisibility, ui => ui.shareDialog.visible, false);
   });
```

Comment:
I think it would be good to add some tests, similar to these ones, to test the `SHARE_DIALOG` actions

Reply:
OK, shareDialog has only 2 (setVisibility and clearDialog) but I'll make sure they're both covered.

Reply:
Oh yeah, it's definitely a little bit silly but silly tests catch silly coding errors later!